app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+
+    if user&.authenticate params[:session][:password]
```

Comment:
&. có tác dụng gì?

Reply:
&. là cách viết gọn của &&, ngoài ra &. chỉ có tác dụng với nil object và hoàn toàn không có tác dụng với false object.